### DISCUSSION TITLE
partybypass

### DISCUSSION THREAD

**Discussion by PiantaMK** [Fedoraware/Fedoraware#1123] [Ideas]
we can paste from cathook

**Comment by Niximkk** [comment]
As a bot hoster i can say :True: 

https://github.com/nullworks/cathook/pull/677/commits/ca9c861e15af851ff21169e687dc760edaefecf0

https://github.com/nullworks/cathook/wiki/What-is-Partybypass-and-how-do-I-use-it%3F

**Comment by Niximkk** [comment]
Btw lbox lobby uses this

**Comment by PiantaMK** [comment]
ik lbox uses this i already read the wiki earlier today

**Comment by SDarkra1** [comment]
@PiantaMK https://github.com/Necres1977/cathook-PartyBypass (Easiest Paste ever (if the offset still works))
